### ISSUE TITLE
firefox: `nixpkgs.config.firefox.enableVLC = true` is broken on 18.03

### ISSUE THREAD

**Issue by knedlsepp** [NixOS/nixpkgs#44930]
## Issue description
When trying to install firefox with `enableVLC`, I get the following error:
```
building '/nix/store/f5bhzr69bdvxp0gp052m2jg4y7fc472i-firefox-61.0.2.drv'...
output '/nix/store/z6y7hh15y8hbdj73jqr5h2hddafdg25b-firefox-61.0.2' is not allowed to refer to the following paths:
    /nix/store/9y2f87qb1djmpjs1gxl6smfkpl581waa-gcc-wrapper-7.3.0
```

It seems `enableVLC` is already gone on `master`: 
https://github.com/NixOS/nixpkgs/commit/3107b4f0187d2470b836e89943b229b6c8cdb098#diff-d546c0cc4fa5e6ccb4772cdeab3c38a8

Considering it doesn't work on 18.03 anyway, could we cherry-pick the commit to 18.03 to disable the VLC functionality?

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.86, NixOS, 18.03.133069.bfeab2f4615 (Impala)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0.4`
 - channels(sepp): `"nixos-17.09-17.09.3204.d16ffa15090"`
 - channels(root): `"nixos-18.03.133069.bfeab2f4615"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by tycho01** [comment]
For whatever reason, on master I seem to be getting this error even without this (or other firefox configurations).
So the issue could be broader than this.
I suppose for you it worked after removing this option then?

**Comment by vcunat** [comment]
This is _now_ caused by another source; it should be fixed by https://github.com/NixOS/nixpkgs/pull/45442 (in `staging` branch, half of binaries built ATM).

**Comment by vcunat** [comment]
The other issue is also fixed on master since 7db611f.
